var/spack/repos/builtin/packages/fastmath/package.py

```diff
@@ -19,13 +19,11 @@ class Fastmath(Package):
     version('1.0', 'e2b724dfcc31d735897971db91be89ff')
 
     # BundlePackage
-    depends_on('boxlib dims=3')
+    depends_on('amrex')  # default is 3 dimensions
```

Comment:
I'm not sure how robust you want fastmath package to be.
for ex: amrex has a conflict with clang. And its not easy to encode this in the metapackage.
xsdk enables amrex only for gcc and intel compiler.

Reply:
Sorry, I resolved this and it didn't take my comment. Not looking for any more robustness than existed prior to this change. I think if it works only on vanilla gnu, that is all that is expected.